Yoast/wordpress-seo
# Remove merging to trunk from rc script

## Context


* To simplify and reduce problems.
Merging back to trunk is an easy task that can be performed afterwards.

## Summary


This PR can be summarized in the following changelog entry:

* [non-user-facing] Removes merging to trunk from the RC steps. Moves the shell commands needed to do GIT actions to grunt-commands.

## Relevant technical choices:

* n/a

## Test instructions

This PR can be tested by following these steps:

* Use the ` grunt bump-rc-version-and-commit --plugin-version=14.5 --type=release` command to bump the versions and see a commit being added to the branch with the bumped versions.

## UI changes

## Documentation

## Quality assurance


Fixes #15417
